src/components/form/form.js

```diff
@@ -529,54 +545,115 @@ class Form extends React.Component {
   }
 
   /**
-   * Gets the save button for the form
+   * The default Save button for the form
+   *
+   * @method defaultSaveButton
+   * @return {Object} JSX
+   */
+  defaultSaveButton = () => {
+    return (
+      <SaveButton
+        saveButtonProps={ this.props.saveButtonProps }
+        saveText={ this.props.saveText }
+        saving={ this.props.saving }
+        errors={ this.state.errorCount }
+        warnings={ this.state.warningCount }
+      />
+    );
+  }
+
+  /**
+   * Returns a custom save button if passed in
+   * the default if not
+   *
    * @method saveButton
-   * @return {Object} JSX save button
+   * @return {Object} JSX
    */
-  get saveButton() {
-    let saveClasses = classNames(
-          "carbon-form__save", {
-            "carbon-form__save--invalid": this.state.errorCount || this.state.warningCount
-          }
-        ),
-        saveProps = assign({}, this.props.saveButtonProps, { as: 'primary', disabled: this.props.saving });
+  saveButton = () => {
+    if (!this.props.save) { return null; }
 
+    return this.props.customSaveButton ? this.props.customSaveButton : this.defaultSaveButton();
+  }
+
+  /**
+   * Returns a form summary
+   *
+   * @method saveButtonWithSummary
+   * @return {Object} JSX
+   */
+  saveButtonWithSummary = () => {
     return (
-      <div className={ saveClasses }>
-        <FormSummary errors={ this.state.errorCount } warnings={ this.state.warningCount } />
-        <Button { ...saveProps } data-element='save'>
-          { this.props.saveText || I18n.t('actions.save', { defaultValue: 'Save' }) }
-        </Button>
+      <FormSummary
+        className='carbon-form__summary'
+        errors={ this.state.errorCount }
+        warnings={ this.state.warningCount }
+      >
+        { this.saveButton() }
+      </FormSummary>
+    );
+  }
+
+  /**
+   * Returns the buttons for the form
+   *
+   * @method buttons
+   * @return {Object} JSX
+   */
+  buttons = () => {
```

Comment:
These references to `buttons` probably ought to be `form-footer`:

```
formFooter = () => {
  return (
    <div className='form__footer'>
      { this.props.showSummary ? this.saveButtonWithSummary() : this.saveButton() }
      { this.cancelButton() }
    </div>
  );
}
```

This will handle the styling of the footer, as per those designs